galsim/interpolant.py

```diff
@@ -100,6 +104,14 @@ def from_name(name, tol=1.e-4, gsparams=None):
     def gsparams(self):
         return self._gsparams
 
+    @property
+    def positive_flux(self):
```

Comment:
These should probably get a docstring.  Maybe just a single sentence description.